cicecore/cicedynB/analysis/ice_history.F90

```diff
@@ -972,7 +997,7 @@ subroutine init_hist (dt)
              "none", c1, c0,                              &
              ns1, f_aisnap)
       
-         call define_hist_field(n_trsig,"trsig","N/m^2",tstr2D, tcstr, &
```

Comment:
Is this correct?  This looks like it's reverting a PR from last week?

Reply:
N/m is correct.  This should be okay, although it was already corrected previously.

Reply:
I think it was part of Dave's PR #198 but I'd changed it here as well. Since this is identical to the current master I'm not sure why it's showing up as changed.

Reply:
I still haven't figured out what git is doing when it shows PR diffs.  Sometimes, it seems like it's comparing to master, other times it seems like it's comparing to the branch base.  I still find it confusing.